api/sql/tripSQL/initialize.sql

```diff
@@ -0,0 +1,14 @@
+DROP TABLE IF EXISTS Trips CASCADE;
+
+CREATE TABLE Trips (
```

Comment:
Need to specify which values cannot be null (i.e. purpose can be null, I believe?)